### DISCUSSION TITLE
WebSocket ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#62] [Announcements]
# WebSocket ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/guides/webdraftt/websocket/

**Comment by Bahurtsev** [comment]
У меня складывается впечатление что я один это учу.) Ни одного комментария, Всем всё понятно у всех работает.) Я вчера потратил 5 часов чтобы у меня заработал первый пример который как бы уже решён и должен работать.) В какой учебник не загляну, написано тоже самое и прикол в том что и у них тоже не работает.) И только у какого-то там американца, которого я нашёл через VPN пример несколько отличается. И только благодаря ему решил.

**Reply by Bahurtsev** [reply]
Исправьте:

http = require('http').createServer(app)

На

http = require('http').server(app)

А так же:

io.clients().server.engine.clientsCount

на:

io.sockets.server.engine.clientsCount

**Reply by SydorRoman** [reply]
@Bahurtsev,  не один)

**Comment by heha1115** [comment]
помогите require(...).server is not a function перебрал все возможные варианты а все равно ничего не происходит
